inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -128,13 +149,14 @@ def simulate_async(self,
         """Simulates the wind tunnel scenario asynchronously.
 
         Args:
+            simulator: Simulators available [OpenFOAM].
             object_path: Path to object inserted in the wind tunnel.
-            simulator: Simulator to use for the simulation.
-            output_dir: Path to the directory where the simulation output
-                is downloaded.
-            simulation_time: Simulation time, in seconds.
-            write_interval: Interval between simulation outputs, in seconds.
+            simulation_time: Simulation time, in seconds (s).
+            output_time_step: Interval between simulation outputs, in seconds (s).
             n_cores: Number of cores to use for the simulation.
+            resolution: Level of detail for the simulation. It can be
+                "high", "medium" or "low".
+            resource_pool_id: Id of the resource pool to use for the simulation.
```

Comment:
I think we should also include a usage example in the documentation with the outputs expected.

Reply:
I will adapt it later when merging sync/async.